bumble/gatt_client.py

```diff
@@ -112,13 +126,13 @@ class ServiceProxy(AttributeProxy):
     characteristics: List[CharacteristicProxy]
     included_services: List[ServiceProxy]
 
-    @staticmethod
-    def from_client(service_class, client, service_uuid):
+    @classmethod
```

Comment:
This change might not always work: the `service_class` passed here may not necessarily be `ServiceProxy` or a subclass.